driver/mloTripletMarginLossHost.hpp

```diff
@@ -0,0 +1,391 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2024 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+
+#pragma once
+
+#include <../test/ford.hpp>
+
+#include <miopen/miopen.h>
+#include <miopen/tensor.hpp>
+#include <miopen/tripletmarginloss/utils.hpp>
+
+template <class T>
+float dist(const T* I1,
+           const T* I2,
+           const int p,
+           const float eps,
+           const tensor_view_t<2> I1_tv,
+           const tensor_view_t<2> I2_tv,
+           const size_t n,
+           const size_t C)
+{
+    // eps defined in
+    // https://pytorch.org/docs/stable/generated/torch.nn.PairwiseDistance.html
```

Comment:
This comment is about the default value of eps. It's not here anymore so this is not the right spot.
And as far as I remember AMD didn't want external web link other than rocm environment to be in their code so I'll just remove it if I were you.

Reply:
I have removed those comments about `eps`.